### DISCUSSION TITLE
What's the right syntax for `cloudinit_runcmd_common`?

### DISCUSSION THREAD

**Discussion by peda** [kube-hetzner/terraform-hcloud-kube-hetzner#691] [Q&A] [answered]
Hello,

by looking through customization options to the nodes themselves, I came across a setting called `cloudinit_runcmd_common`

What's the correct syntax for this field? Is it a multiline string or is it an array? Or is it something completely different?

Thanks in advance,
Peter

**Comment by mysticaltech** [comment, marked as answer]
@peda Just a multi-line string used in the cloud-config templates.
